src/App.tsx

```diff
@@ -21,6 +32,10 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={perPage}
+            onChange={event => {
+              handleValueChange(+event.target.value);
+            }}
```

Comment:
it is bad practise to make anonymous arrow function in jsx
make handler  and pass reference here